doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -314,6 +314,9 @@ nRF Desktop
   * ``Error while sending report`` log level in :ref:`nrf_desktop_hid_state` from error to warning.
     The log might appear, for example, during the disconnection of a HID transport.
     In that case, it does not denote an actual error.
+  * Updated number of ATT buffers (:kconfig:option:`CONFIG_BT_ATT_TX_COUNT`) for nRF Desktop peripherals.
+    The number of buffers allows peripherals to simultaneously send all of the supported HID notifications (including pipeline support), the BAS notification and an ATT response.
+    ATT uses a separate net buffer pool.
```

Comment:
Could you specify here to what it is separate from? Or maybe this sentence is not needed?

Reply:
separate -> dedicated?

Reply:
dedicated seems better